app/setup_pf.py

```diff
@@ -116,6 +116,8 @@ def setup_configs():
         seek_query_form_custom_htmls={'en': '', 'fr': ''},
         published_date=get_timestamp(datetime(2010, 1, 12)),
         updated_date=get_timestamp(datetime(2010, 1, 12)),
+        admin_user_email=get_current_user_email(),
```

Comment:
If you intend these to be global configs, you should initialize them in config.set(...) above instead.